test/jdk/java/awt/print/PrinterJob/InvalidPage.java

```diff
@@ -21,223 +21,121 @@
-
-      messageText = new TextArea( "", 5, maxStringLength, scrollBoth );
-      add("Center", messageText);
-
-      pack();
-
-      show();
-    }// TestDialog()
-
-   //DO NOT call this directly, go through Sysout
-   public void printInstructions( String[] instructions )
-    {
-      //Clear out any current instructions
-      instructionsText.setText( "" );
-
-      //Go down array of instruction strings
-
-      String printStr, remainingStr;
-      for( int i=0; i < instructions.length; i++ )
-       {
-         //chop up each into pieces maxSringLength long
-         remainingStr = instructions[ i ];
-         while( remainingStr.length() > 0 )
-          {
-            //if longer than max then chop off first max chars to print
-            if( remainingStr.length() >= maxStringLength )
-             {
-               //Try to chop on a word boundary
-               int posOfSpace = remainingStr.
-                  lastIndexOf( ' ', maxStringLength - 1 );
-
-               if( posOfSpace <= 0 ) posOfSpace = maxStringLength - 1;
-
-               printStr = remainingStr.substring( 0, posOfSpace + 1 );
-               remainingStr = remainingStr.substring( posOfSpace + 1 );
-             }
-            //else just print
-            else
-             {
-               printStr = remainingStr;
-               remainingStr = "";
-             }
-
-            instructionsText.append( printStr + "\n" );
-
-          }// while
-
-       }// for
-
-    }//printInstructions()
-
-   //DO NOT call this directly, go through Sysout
-   public void displayMessage( String messageIn )
-    {
-      messageText.append( messageIn + "\n" );
+    private static final String INSTRUCTIONS =
+            " You must have a printer available to perform this test\n" +
+            " Press the print button, which brings up a print dialog and\n" +
+            " in the dialog select a printer and press the print button\n" +
+            " in the dialog. Repeat for as many printers as you have installed\n" +
+            " On solaris and linux just one printer is sufficient\n" +
+            " Collect the output and examine it, each print job has two pages\n" +
+            " of very similar output, except that the 2nd page of the job may\n" +
+            " appear in a different colour, and the output near the edge of\n" +
+            " the page may be clipped. This is OK. Hold up both pieces of paper\n" +
+            " to the light and confirm that the lines and text (where present)\n" +
+            " are positioned identically on both pages\n" +
+            " The test fails if the JRE crashes, or if the output from the two\n" +
+            " pages of a job is aligned differently";
+
+    public static void main(String[] args) throws Exception {
+
+        if (PrinterJob.lookupPrintServices().length == 0) {
+            throw new SkippedException("Printer not configured or available."
+                    + " Test cannot continue.");
+        }
```

Comment:
```suggestion
    public static void main(String[] args) throws Exception {
        if (PrinterJob.lookupPrintServices().length == 0) {
            throw new SkippedException("Printer not configured or available.");
        }
```
Remove the blank line; shorten the message.